receiver/mongodbatlasreceiver/alerts_test.go

```diff
@@ -569,3 +580,196 @@ func compareLogRecord(expected, actual plog.LogRecord) error {
 
 	return nil
 }
+
+const (
+	testAlertID         = "633335c99998645b1803c60b"
+	testGroupID         = "5bc762b579358e3332046e6a"
+	testProjectID       = "test-project-id"
+	testProjectName     = "test-project"
+	testMetricName      = "metric-name"
+	testTypeName        = "OUTSIDE_METRIC_THRESHOLD"
+	testHostNameAndPort = "127.0.0.1:27017"
+	testClusterName     = "Cluster1"
+)
+
+func TestAlertsRetrieval(t *testing.T) {
+	cases := []struct {
+		name            string
+		config          func() *Config
+		client          func() alertsClient
+		validateEntries func(*testing.T, plog.Logs)
+	}{
+		{
+			name: "default",
+			config: func() *Config {
+				return &Config{
+					ScraperControllerSettings: scraperhelper.NewDefaultScraperControllerSettings(typeStr),
+					Granularity:               defaultGranularity,
+					RetrySettings:             exporterhelper.NewDefaultRetrySettings(),
+					Alerts: AlertConfig{
+						Mode: alertModePoll,
+						Projects: []ProjectConfig{
+							{
+								Name: testProjectName,
+							},
+						},
+						PollInterval: 1 * time.Second,
+					},
+				}
+			},
+			client: func() alertsClient {
+				return testClient()
+			},
+			validateEntries: func(t *testing.T, logs plog.Logs) {
+				expectedStringAttributes := map[string]string{
+					"id":           testAlertID,
+					"status":       "TRACKING",
+					"event.domain": "mongodbatlas",
+					"metric.name":  testMetricName,
+					"type_name":    testTypeName,
+				}
+				validateAttributes(t, expectedStringAttributes, logs)
+			},
+		},
+		{
+			name: "project cluster inclusions",
+			config: func() *Config {
+				return &Config{
+					ScraperControllerSettings: scraperhelper.NewDefaultScraperControllerSettings(typeStr),
+					Granularity:               defaultGranularity,
+					RetrySettings:             exporterhelper.NewDefaultRetrySettings(),
+					Alerts: AlertConfig{
+						Mode: alertModePoll,
+						Projects: []ProjectConfig{
+							{
+								Name:            testProjectName,
+								IncludeClusters: []string{testClusterName},
+							},
+						},
+						PollInterval: 1 * time.Second,
+					},
+				}
+			},
+			client: func() alertsClient {
+				return testClient()
+			},
+			validateEntries: func(t *testing.T, logs plog.Logs) {
+				require.Equal(t, logs.LogRecordCount(), 1)
+			},
+		},
+	}
+
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			logSink := &consumertest.LogsSink{}
+			alertsRcvr, err := newAlertsReceiver(zap.NewNop(), tc.config(), logSink)
+			require.NoError(t, err)
+			alertsRcvr.client = tc.client()
+			alertsRcvr.storageClient = storage.NewNopClient()
```

Comment:
This shouldn't be necessary, as it is baked into `adapter.GetStorageClient`, which is called in `Start`